src/utils/sigma.utils.js

```diff
@@ -608,6 +626,45 @@
   };
 
   /**
+   * Extract the center from a mouse or touch event.
+   *
+   * @param  {event}  e A mouse or touch event.
+   * @return {object}   The center of the event's target.
+   */
+  sigma.utils.getCenter = function(e) {
+    var ratio = e.target.getAttribute('class') == 'sigma-svg' ? 1 :
```

Comment:
Getting the class is dangerous if the user add some classes of its own. Furthermore, if I recall correctly, the `sigma-` prefix in the class is actually a setting.